packages/react-components/src/components/ImageOverlay.tsx

```diff
@@ -155,9 +155,8 @@ export const ImageOverlay = (props: ImageOverlayProps): JSX.Element => {
             alt={altText || 'image'}
             aria-label={'image-overlay-main-image'}
             aria-live={'polite'}
-            onError={(event) => {
+            onError={() => {
               setIsImageLoaded(false);
-              onError && onError(event);
```

Comment:
why is OnError removed?

Reply:
> why is OnError removed?

We decided not to expose that API yet since the chance for Contoso needing this API is very low. We'll keep the API simple for now unless there's customer ask.